src/app/qgisapp.cpp

```diff
@@ -13429,8 +13429,20 @@ QgsRasterLayer *QgisApp::addRasterLayerPrivate(
     refreshBlocker = qgis::make_unique< QgsCanvasRefreshBlocker >();
   }
 
+  QString shortName = name;
+  QRegularExpression reRasterFile( "^/vsi(.+/)*([^ ]+)( .+)?$", QRegularExpression::CaseInsensitiveOption );
+  QRegularExpressionMatch matchRasterFile = reRasterFile.match( name );
+
+  if ( matchRasterFile.hasMatch() )
+  {
+    if ( matchRasterFile.captured( 2 ).length() > 5 )
```

Comment:
Why greater than five here? E.g., m.tif is five characters. I'd merely insure that the captured group isn't empty.